src/filter.js

```diff
@@ -0,0 +1,33 @@
+const curry = require('./curry')
+const filterTransformer = require('../transformers/filter')
+const getAddReducer = require('../internal/getAddReducer')
+const getSeed = require('../internal/getSeed')
```

Comment:
Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you